babel/util.py

```diff
@@ -217,6 +218,12 @@ def wraptext(text: str, width: int = 70, initial_indent: str = '', subsequent_in
     :param subsequent_indent: string that will be prepended to all lines save
                               the first of wrapped output
     """
+    warnings.warn(
+        "babel.util.wraptext is deprecated and will be removed in a future version of Babel. "
+        "If you need this functionality, use the `babel.util.TextWrapper` class directly.",
+        PendingDeprecationWarning,
```

Comment:
Wouldn't a normal `DeprecationWarning` be more appropriate here, since this would be a normal deprecation and not a pending deprecation?